lib/chartmogul/api/actions/update.rb

```diff
@@ -15,6 +19,22 @@ def update!
 
           assign_all_attributes(json)
         end
+
+        module ClassMethods
+          def update!(uuid, attributes = {})
```

Comment:
This is good, but your example is incorrect (UUID passed differently).
I also assume this will be now on more classes, can you add tests for those, too, please?